clang/runtime/dpct-rt/include/dnnl_utils.hpp.inc

```diff
@@ -155,23 +155,20 @@ public:
            memory_format_tag *tag, int *ndims, int dims[]) const;
   /// Getting dims from a ND memory.
   /// \return The dims.
-  std::vector<int64_t> get_dims() const { return _desc.dims(); }
+  std::vector<int64_t> get_dims() const { return _desc.get_dims(); }
   /// Getting strides from a ND memory.
   /// \return The strides.
   std::vector<int64_t> get_strides() const {
-    int ndims = _desc.data.ndims;
-    return std::vector<int64_t>(_desc.data.format_desc.blocking.strides,
-                                _desc.data.format_desc.blocking.strides +
-                                    ndims);
+    return _desc.get_strides();
   }
   /// Getting element num from a ND memory.
   /// \return The element number.
   size_t get_element_num() const {
-    if (_desc.dims().empty()) {
+    if (_desc.get_dims().empty()) {
```

Comment:
How about use a variable like "dims = _desc.get_dims()" since it is also used after in a loop.